lang/en.js

```diff
@@ -230,7 +230,7 @@
     "constants.plans.pro": "Pro plan",
     "constants.plans.biz": "Business plan",
     "constants.plans.ent": "Enterprise plan",
-    "errors.noActiveZoneSelected": "Please select a domain that is provisioned with Cloudflare.",
+    "errors.noActiveZoneSelected": "Please continue to {link} to set up this domain.",
```

Comment:
Can we update this to:
```
It looks like {domain} isn't provisioned with Cloudflare.  Please continue to {link} to provision this domain.
```

This should help us debug the WordPress issues where they have a subdomain or a folder.